connectors/src/connectors/slack/chat/stream_conversation_handler.ts

```diff
@@ -185,13 +155,6 @@ async function streamAgentAnswerToSlack(
           )
         );
       }
-      case "error": {
```

Comment:
@philipperolet as author, do you know where this is supposed to come from ? AFAICT this isn't actually an event that we stream, but could be wrong